docs/UserMenu.md

```diff
@@ -0,0 +1,107 @@
+# User Menu
+
+The `<pxb-user-menu>` is an Avatar that opens a Menu when clicked. It is typically used in the top-right corner of an application and indicates who is logged in.
+
+<div style="align-items: center; display:flex; justify-content: space-around">
+
+<img width="30%" alt="UserMenu Avatar" src="./images/userMenuAvatar.png">
+<img width="35%" alt="UserMenu Opened" src="./images/userMenuOpened.png">
+
+</div>
+
+The Menu can be populated via the `menuGroups` prop, or can be entirely customized by supplying your own content.  See the API section below for more details.
```

Comment:
Do we need to tweak the description up here? I didn't see a menuGroups prop on the Angular user-menu.

Reply:
Yes this was leftover from when we still created the menu body from the menuGroups prop.  I've updated it.